test/fw/ptl/utils/plugins/ptl_test_loader.py

```diff
@@ -190,7 +196,19 @@ def prepareTestLoader(self, loader):
         old_loadTestsFromNames = loader.loadTestsFromNames
 
         def check_loadTestsFromNames(names, module=None):
-            rv = old_loadTestsFromNames(names, module)
+            tests_dir = names
+            if not self.testfiles:
+                ptl_test_dir = __file__
+                ptl_test_dir = os.path.join(ptl_test_dir.split('ptl')[0],
```

Comment:
don't you also need to modify the Makefiles so that tests get copied to this path when PTL is installed?

Reply:
We are assuming here that when PTL is installed, tests will reside in parallel to PTL library.

Reply:
> don't you also need to modify the Makefiles so that tests get copied to this path when PTL is installed?

This is already done. see https://github.com/PBSPro/pbspro/blob/master/test/tests/Makefile.am